WordPress/Classes/ViewRelated/Reader/WPStyleGuide+Reader.swift

```diff
@@ -289,15 +289,40 @@ extension WPStyleGuide {
         applyReaderActionButtonStyle(button)
     }
 
-    @objc public class func applyReaderSaveForLaterButtonTitles(_ button: UIButton) {
-        let saveTitle = WPStyleGuide.savePostStringForDisplay(false)
-        let savedTitle = WPStyleGuide.savePostStringForDisplay(true)
+    /// Applies the save for later button style to the button passed as an argument
+    /// - Parameter button: the button to apply the style to
+    /// - Parameter showTitle: if set to true, will show the button label (default: true)
+    @objc public class func applyReaderSaveForLaterButtonTitles(_ button: UIButton, showTitle: Bool = true) {
+        let saveTitle = showTitle ? WPStyleGuide.savePostStringForDisplay(false) : ""
+        let savedTitle = showTitle ? WPStyleGuide.savePostStringForDisplay(true) : ""
 
         button.setTitle(saveTitle, for: .normal)
         button.setTitle(savedTitle, for: .selected)
         button.setTitle(savedTitle, for: [.highlighted, .selected])
     }
 
+    /// Applies the reblog button style to the button passed as an argument
+    /// - Parameter button: the button to apply the style to
+    /// - Parameter showTitle: if set to true, will show the button label (default: true)
+    @objc public class func applyReaderReblogActionButtonStyle(_ button: UIButton, showTitle: Bool = true) {
+        let size = Gridicon.defaultSize
+        let icon = Gridicon.iconOfType(.reblog, withSize: size)
+
+        button.setImage(icon, for: .normal)
+
+        WPStyleGuide.applyReaderReblogActionButtonTitle(button, showTitle: showTitle)
+        WPStyleGuide.applyReaderActionButtonStyle(button)
+    }
+
+    /// Applies the reblog button title to the button passed as an argument
+    /// - Parameter button: the button to apply the title to
+    /// - Parameter showTitle: if set to true, will show the button label (default: true)
+    @objc public class func applyReaderReblogActionButtonTitle(_ button: UIButton, showTitle: Bool = true) {
+        let title = showTitle ? NSLocalizedString("Reblog", comment: "Text for the 'Reblog' button.") : ""
```

Comment:
Should this fall back to "Reblog" instead of ""? It might be better to show _something_ in case the translation fails.

Reply:
This method will set the title to an empty string if `showTitle` is false, which means we want the button to drop the label. `NSLocalizedString` should fall back to its key ("Reblog" in this case) if no translation is found. The method documentation in the comment is not very clear as it is, I'll add some clarifications

Reply:
My apologies! I misread what this method was doing.